include/mitsuba/core/bitmap.h

```diff
@@ -239,6 +239,10 @@ class MTS_EXPORT_CORE Bitmap : public Object {
     /// Move constructor
     Bitmap(Bitmap &&bitmap);
 
+
+    int getBytesPerComponent() const;
```

Comment:
`underscore_case` convention
```suggestion
    int get_bytes_per_component() const;
```